FAQ: Can the party that calls SPC validate the assertion even if not the RP? (#36)
w3c/secure-payment-confirmation: README.md

[Web Authentication]: https://www.w3.org/TR/webauthn/
[pilot]: https://bit.ly/webauthn-to-pay-2020h2-pilot

## FAQ

> ### Q. Can the party that calls SPC validate the assertion even if not the RP?

Reviewer: ```suggestion
### Q. Who can validate the SPC assertion besides the actual Relying Party (RP)?
```

> Yes. The Relying Party can share the credentialPublicKey (of the authenticator data) with another party, e.g., through an out-of-band communication.

Reviewer: ```suggestion
Validation in the context of SPC means the ability to verify that the signature in the SPC response is a signature over the SPC challenge (which bundles together transaction details with transaction-specific dynamic data from the RP). An RP can choose to share the credential public key with another party (e.g. the card network or PSP) via out-of-band communication to enable said party to validate the SPC assertion.
```
